### DISCUSSION TITLE
Add Default Action: swift-doc

### DISCUSSION THREAD

**Discussion by yhkaplan** [fastlane/fastlane#18399] [Ideas]
Next to Jazzy, [swift-doc](https://github.com/SwiftDocOrg/swift-doc) is quickly becoming a popular alternative tool to generate documentation for Swift projects. It already has 1.5k stars on GitHub, and due to the fact that it doesn't require building the project before generating documentation, it's quite fast and will likely continue to grow. 

If you're interested in this, I will make a default action, otherwise making a plugin I guess.

**Comment by joshdholtz** [comment]
@yhkaplan If you want to make a plugin for this that would be great! 😊

**Reply by yhkaplan** [reply]
Will do, thanks!

**Reply by yhkaplan** [reply]
Done! https://github.com/yhkaplan/fastlane-plugin-swift_doc

**Reply by yhkaplan** [reply]
@joshdholtz Quick question, how do I get my plugin listed in the available plugin list? https://docs.fastlane.tools/plugins/available-plugins/

Is it crawled automatically?

**Reply by minuscorp** [reply]
It is!

**Reply by yhkaplan** [reply]
Neat! Thanks☺️
